### PR TITLE
hero_section

### PR DESCRIPTION
**Pull request by Wawrzyni3c**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -47,5 +47,80 @@
...
+                >DNA</a
+              >
+              Test That Provides Truly Personalized Insights Based on Your
+              Genetic Results
+            </h1>
+          </div>
+
+          <div class="hero_frame">
+            <div class="frame_text">
+              <img src="./public/icons/Tick Square.png" />
```
Comment (EnDriash) [inline index.html]: icons should be in <i /> element

File: scss/sections/_hero.scss [hunk 2]
```diff
@@ -0,0 +1,220 @@
+#hero {
+  display: flex;
+  max-height: 1080px;
+  width: 90%;
+  margin: 0 0 0 10%;
+  flex-direction: row;
+  justify-content: space-between;
+
+  .hero_text {
```
Comment (EnDriash) [inline scss/sections/_hero.scss]: eeeeee title is more fit here.
text is to more generic meaning

File: scss/sections/_hero.scss [hunk 3]
```diff
@@ -0,0 +1,220 @@
...
+        }
+      }
+    }
+    .button-price {
+      display: flex;
+      width: 80%;
+      margin: 3% 0;
+      flex-direction: row;
+
+      .submit_button {
```
Comment (EnDriash) [inline scss/sections/_hero.scss]: all buttons you should move to another file in scss/components/buttons.scss

File: scss/sections/_informations.scss [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+      }
+    }
+
+    .movie {
+      width: 100%;
+    }
+  }
+}
+
+@media (min-width: 992px) and (max-width: 1199px) {
```
Comment (EnDriash) [inline scss/sections/_informations.scss]: if you dont need it please remove it.

File: scss/sections/_informations.scss [hunk 5]
```diff
@@ -0,0 +1,80 @@
...
+  .movie {
+    width: 50%;
+
+    video {
+      width: 100%;
+      border-radius: 10px 10px 0 0;
+      margin-top: 3rem;
+    }
+  }
+}
```
Comment (EnDriash) [inline scss/sections/_informations.scss]: spaces!! between blocks!

File: index.html [hunk 6]
```diff
@@ -47,5 +47,80 @@
...
+              <p>Custom health pln builder</p>
+            </div>
+          </div>
+
+          <div class="button-price">
+            <button class="submit_button">
+              order now<img src="./public/icons/Arrow - Right Circle.png" />
+            </button>
+            <div class="price">
```
Comment (EnDriash) [inline index.html]: price_block?

File: index.html [hunk 7]
```diff
@@ -47,5 +47,80 @@
...
+              <div class="only_for">
```
Comment (EnDriash) [inline index.html]: are you sure that you need this div here?

File: index.html [hunk 8]
```diff
@@ -47,5 +47,80 @@
...
+                <p>Only For</p>
+              </div>
+              <div class="money">
```
Comment (EnDriash) [inline index.html]: money_block

File: index.html [hunk 9]
```diff
@@ -47,5 +47,80 @@
...
+              <img src="./public/icons/Tick Square.png" />
+              <p>40+ personalized wellness reports</p>
+            </div>
+            <div class="frame_text">
+              <img src="./public/icons/Tick Square.png" />
+              <p>Custom health pln builder</p>
+            </div>
+          </div>
+
+          <div class="button-price">
```
Comment (EnDriash) [inline index.html]: so for naming do you use - or _?? keep consistent !
additionaly if you would like to use BEM metodic it should be
Block_Element-Modifier so
hero_button-price
if you use OOCSS
you should just call it
price

additionaly
why you have so weird nesting construction ?

**Review by EnDriash — CHANGES_REQUESTED** [review]

File: scss/sections/_hero.scss (hunk 2 above)
Comment (Wawrzyni3c) [reply scss/sections/_hero.scss]: can i replace "text" with "title"?

File: scss/sections/_informations.scss (hunk 5 above)
Comment (Wawrzyni3c) [reply scss/sections/_informations.scss]: can you describe it more?

File: index.html (hunk 1 above)
Comment (Wawrzyni3c) [reply index.html]: can you describe it more?

**Comment by EnDriash** [comment]
You have conflicts to review

**Review by EnDriash — CHANGES_REQUESTED** [review]
Still conflicts to review

File: scss/index.css [hunk 10]
```diff
@@ -10,7 +10,7 @@
   flex: 100%;
 }
 
-[class*=col-] {
+[class*="col-"] {
```
Comment (EnDriash) [inline scss/index.css:13]: Gku sure if this work.

Comment (EnDriash) [reply scss/index.css:13]: Are you?

**Review by EnDriash — APPROVED** [review]

**Review by EnDriash — CHANGES_REQUESTED** [review]
Still you didnt resolved conflicts

**Review by EnDriash — APPROVED** [review]

**Pull request merged**